### ISSUE TITLE
make background transparent

### ISSUE THREAD

**Issue by edigonzales** [sogis/docker-mapcache#16]
Is there a reason we do not have a transparent background? Can we try to add the transparent=true (since we use png) query param in the mapcache.xml?

**Comment by edigonzales** [comment]
https://github.com/geoadmin/mf-geoadmin3/pull/4720

**Comment by schmandr** [comment]
> 
> 
> Is there a reason we do not have a transparent background? Can we try to add the transparent=true (since we use png) query param in the mapcache.xml?

Yes, let's try it. There is no real reason. It's just because the examples in the MapCache docs usually just mention the FORMAT and LAYERS params.

**Comment by schmandr** [comment]
OK, result is better than before, but more work is to be done.

Result with test WMTS black/white: https://mf-geoadmin3.int.bgdi.ch/wmts-kanton-solothurn/index.html?topic=ech&lang=de&bgLayer=ch.swisstopo.pixelkarte-farbe&layers=ch.swisstopo.zeitreihen,ch.bfs.gebaeude_wohnungs_register,ch.bav.haltestellen-oev,ch.swisstopo.swisstlm3d-wanderwege,WMTS%7C%7Cch.so.agi.hintergrundkarte_sw%7C%7Chttps:%2F%2Fgeo-t.so.ch%2Fapi%2Fwmts%2F1.0.0%2FWMTSCapabilities.xml&layers_visibility=false,false,false,false,true&layers_timestamp=18641231,,,,&E=2555829.01&N=1214797.82&zoom=1
